cpp/mg_utility/mg_utils.hpp

```diff
@@ -0,0 +1,209 @@
+/// @file mg_utils.hpp
+///
+/// The file contains methods that connect mg procedures and the outside code
+/// Methods like mapping a graph into memory or assigning new mg results or
+/// their properties are implemented.
+#pragma once
+
+#include <functional>
+
+#include "mg_graph.hpp"
+#include "mg_procedure.h"
+
+namespace mg_graph {
+
+template <typename TSize>
+void CreateGraphNode(mg_graph::Graph<TSize> *graph, const mgp_vertex *vertex) {
+  // Get Memgraph internal ID property
+  auto id_val = mgp_vertex_get_id(vertex);
+  TSize memgraph_id = id_val.as_int;
```

Comment:
You mean auto?
Because this will trim to the TSize and not use the memgraphs uint64_t as it should. Be careful now what is of which type.

Reply:
Great catch. Need to check twice that part of the code again before merging.